.buildkite/steps/deploy-unit_test.sh

```diff
@@ -5,12 +5,16 @@ set -euo pipefail
 docker stop mongo || true
 docker rm mongo || true
 docker volume rm cyberway-mongodb-data || true
+docker volume rm cyberway-nodeos-data || true
```

Comment:
Why do you need `cyberway-nodeos-data` volume for unit-tests? As I know unit-tests runs independently without `nodeos`.

Reply:
Ok. These need for correct execution when `deploy-unit_test.sh` run on new agent.